src/abstractdataframe/io.jl

```diff
@@ -184,8 +184,13 @@ end
         write(io, "<tr>")
         write(io, "<th>$row</th>")
         for column_name in cnames
-            cell = string(df[row, column_name])
+          if isnull(df[row, column_name])
```

Comment:
You don't need this. Just call [`ourshowcompact`](https://github.com/amellnik/DataFrames.jl/blob/baeb0206fa3e0ac87fecd04d4fd48c5161e0d854/src/abstractdataframe/show.jl#L64) just like (`show`)[https://github.com/amellnik/DataFrames.jl/blob/baeb0206fa3e0ac87fecd04d4fd48c5161e0d854/src/abstractdataframe/show.jl#L255] does. Note in particular the `try...catch` block for undef entries.

Also, please use four-space indents just like in the rest of the code.